### ISSUE TITLE
PHP Notice:  Undefined offset: 1

### ISSUE THREAD

**Issue by aldobel** [Gemorroj/M3uParser#23]
Hi, I have a problem with this package...this is the error log:
`PHP Notice:  Undefined offset: 2 in /vendor/gemorroj/m3u-parser/src/Tag/ExtInf.php on line 91

PHP Notice:  Undefined offset: 1 in /vendor/gemorroj/m3u-parser/src/Tag/ExtInf.php on line 87 `.

Help me please 😊

**Comment by aldobel** [comment]
can anyone help me?

**Comment by sdiama** [comment]
I have the same problem for some m3u8 files, such as:
```
#EXTM3U
#EXT-X-TARGETDURATION:6
#EXT-X-VERSION:3
#EXT-X-MEDIA-SEQUENCE:35620
#EXT-X-PROGRAM-DATE-TIME:2020-12-05T08:37:11Z
#EXTINF:5.279,
2020/12/05/08/37/11-05280.ts
#EXTINF:5.280,
2020/12/05/08/37/17-05280.ts
#EXTINF:5.280,
2020/12/05/08/37/22-05280.ts
#EXTINF:5.280,
2020/12/05/08/37/27-05280.ts
```

The "funny" part is that is parsed correctly at the development environment (Windows) but I get the error at the production server (Linux)

**Comment by Gemorroj** [comment]
Hi! I'm very sorry for this delay. I forgot about this issue =(
Now it fixed in release [5.2.0](../releases/tag/5.2.0)
